azure-pipelines.bundlesize.yml

```
@@ -41,6 +41,39 @@ jobs:
           PathtoPublish: 'apps/test-bundles/dist'
           ArtifactName: distdrop-react
 
+  - job: build_converged
+    timeoutInMinutes: 75
+    pool:
+      vmImage: 'windows-2019'
```

```suggestion
      vmImage: 'ubuntu-20.04'
```

In #17582, I proposed to switch to Ubuntu VMs, it will be nice to update this one, too.